compiler-rt/lib/profile/GCDAProfiling.c

```diff
@@ -584,7 +584,7 @@ void llvm_reset_counters(void) {
   }
 }
 
-#if !defined(_WIN32)
+#if !defined(_WIN32) && !defined(__wasi__)
```

Comment:
This probably makes sense to make `__wasm__` instead of just `__wasi__` since Emscripten also lacks fork.

Reply:
It makes sense to me, thanks!